plugins/modules/zos_job_submit.py

```diff
@@ -10,7 +10,7 @@
 
 ANSIBLE_METADATA = {
     "metadata_version": "1.1",
-    "status": ["stableinterface"],
+    "status": ["preview"],
```

Comment:
Unbeknown to you, @blakeinate took everything out of preview and standardized on `stableinterface`; this is not a big deal what you use here as in Ansible 2.10 this block will be moot and once that becomes the defacto we will remove these METADATA blocks. 

What is concerning is for the same reason below, how old a version are you working off of, because this change was done over 3 months ago.